charts/owncloud/values.yaml

```diff
@@ -25,20 +25,6 @@ owncloud:
   volume_files: /mnt/data/files
   volume_root: /mnt/data
   configExtra: {}
-  # configExtra:
```

Comment:
Would be good if we could keep this as example

Reply:
We have documented it now dont like to duplicate it.

Reply:
Ok